fixed-requirements.txt

```diff
@@ -44,8 +44,8 @@ six==1.13.0
 sseclient-py==1.7
 stevedore==1.30.1
 tooz==1.66.1
-# Note: We use latest version of virtualenv which uses pip 19
-virtualenv==16.6.0
+# Note: We use latest version of virtualenv which uses pip 20
```

Comment:
Would be nice to explicitly list here the exact `pip`, `wheels` and `setuptools` version this virtualenv package provides.
This will make it easier to pin to specific versions in other places I'll show next.